setup.cfg

```diff
@@ -19,6 +19,6 @@ line_length = 79
 multi_line_output = 0
 known_standard_library = pkg_resources,setuptools
 known_first_party = mmaction
-known_third_party = cv2,joblib,matplotlib,mmcv,numpy,pandas,pytest,scipy,seaborn,torch,torchvision
+known_third_party = cudashift,cv2,joblib,matplotlib,mmcv,numpy,pandas,pytest,scipy,seaborn,torch,torchvision
```

Comment:
is cudashift a third party?

Reply:
I can not get rid of it. : )

Reply:
it is an extension. extension does not need to be wrapped as a third-party lib, right? i guess mmdet has lots of extensions